sys-devel/llvm/llvm17-17.0.6.recipe

```diff
@@ -548,6 +548,7 @@ BUILD()
 		$cmakeDirArgs \
 		-DCMAKE_BUILD_TYPE=Release \
 		$cmakeFlags \
+		-DCMAKE_INSTALL_BINDIR=bin \
```

Comment:
@davidkaroly binaries are installed in $prefix/bin on 32bit here, I used the one you did in the PR for LLVM18, this fixed 32bit issues.